scripts/setup_linux.sh

```diff
@@ -1,61 +1,254 @@
-  echo -e "\e[0;31mError: MuJoCo couldn't be set up\e[0m"
-  exit 1
+
+# THE DEFAULTS INITIALIZATION - POSITIONALS
+_positionals=()
+# THE DEFAULTS INITIALIZATION - OPTIONALS
+_arg_tf_gpu="off"
+
+print_help ()
+{
+  printf '%s\n' "Installer of garage for Linux."
+  printf 'Usage: %s [--(no-)tf-gpu] [--(no-)set-envvar] ' "$0"
+  printf '[-h|--help] <mjkey-path>\n' "$0"
+  printf '\t%s\n' "<mjkey-path>: Path of the MuJoCo key"
+  printf '\t%s' "--tf-gpu,--no-tf-gpu: Install TensorFlow GPU instead of the "
+  printf '%s\n' "regular version (off by default)"
+  printf '\t%s' "--set-envvar,--no-set-envvar: Set environment variables "
+  printf '%s\n' "required by garage in .bashrc (off by default)"
+  printf '\t%s\n' "-h,--help: Prints help"
+}
+
+parse_commandline ()
+{
+  while test $# -gt 0
+  do
+    _key="$1"
+    case "$_key" in
+      --no-tf-gpu|--tf-gpu)
+        _arg_tf_gpu="on"
+        test "${1:0:5}" = "--no-" && _arg_tf_gpu="off"
+        ;;
+      --no-set-envvar|--set-envvar)
+        _arg_set_envvar="on"
+        test "${1:0:5}" = "--no-" && _arg_set_envvar="off"
+        ;;
+      -h|--help)
+        print_help
+        exit 0
+        ;;
+      -h*)
+        print_help
+        exit 0
+        ;;
+      *)
+        _positionals+=("$1")
+        ;;
+    esac
+    shift
+  done
+}
+
+
+handle_passed_args_count ()
+{
+  _required_args_string="'mjkey-path'"
+  error_msg="$(echo "FATAL ERROR: Not enough positional arguments - we" \
+    "require exactly 1 (namely: $_required_args_string), but got only" \
+    "${#_positionals[@]}.")"
+  test ${#_positionals[@]} -ge 1 || _PRINT_HELP=yes die "${error_msg}" 1
+  error_msg="$(echo "FATAL ERROR: There were spurious positional arguments" \
+    "--- we expect exactly 1 (namely: $_required_args_string), but got" \
+    "${#_positionals[@]} (the last one was: '${_positionals[*]: -1}').")"
+  test ${#_positionals[@]} -le 1 || _PRINT_HELP=yes die "${error_msg}" 1
+}
+
+assign_positional_args ()
+{
+  _positional_names=('_arg_mjkey_path' )
+
+  for (( ii = 0; ii < ${#_positionals[@]}; ii++))
+  do
+    error_msg="$(echo "Error during argument parsing, possibly an Argbash" \
+      "bug.")"
+    eval "${_positional_names[ii]}=\${_positionals[ii]}" || \
+      die "${error_msg}" 1
+  done
+}
+
+parse_commandline "$@"
+handle_passed_args_count
+assign_positional_args
+### END OF CODE GENERATED BY Argbash (sortof) ### ])
+
+# Verify there's a file in the mjkey path
+test -f "${_arg_mjkey_path}" || _PRINT_HELP=yes die \
+  "The path ${_arg_mjkey_path} of the MuJoCo key is not valid." 1
+
+# Make sure that we're under the garage directory
+GARAGE_DIR="$(readlink -f $(dirname $0)/..)"
+cd "${GARAGE_DIR}"
+
+# File where environment variables are stored
+BASH_RC="${HOME}/.bashrc"
+
+# Install dependencies
+printf "Installing garage dependencies\n"
+printf "You will probably be asked for your sudo password\n"
+sudo apt-get -y -q update
+# To install garage
+sudo apt-get install -y \
```

Comment:
* Install all the apt dependencies in a single command, which is faster and risks fewer conflicts
* User `apt` instead of `apt-get` (which is the new default and has more sophisticated conflict resolution)